extensions/barrage/src/main/java/io/deephaven/extensions/barrage/table/BarrageTable.java

```diff
@@ -301,8 +313,11 @@ private synchronized void realRefresh() {
         localPendingUpdates.clear();
 
         if (update != null) {
-            maybeEnablePrevTracking();
```

Comment:
We moved this elsewhere. It's easier to verify correctness that way, but we're doing prev tracking for one extra (coalesced) message.